{
	"id": "oval:org.debian:def:260591101911682681201207827496578447548",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2018-25091 python-urllib3",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 11",
			"product": "python-urllib3"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2018-25091",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-25091"
			}
		],
		"description": "urllib3 before 1.24.2 does not remove the authorization HTTP header when following a cross-origin redirect (i.e., a redirect that differs in host, port, or scheme). This can allow for credentials in the authorization header to be exposed to unintended hosts or transmitted in cleartext. NOTE: this issue exists because of an incomplete fix for CVE-2018-20060 (which was case-sensitive).",
		"debian": {}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:14850",
								"comment": "python-urllib3 DPKG is earlier than 1.25.6-4"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 11 is installed"
			}
		]
	}
}
